src/main/frontend/db/rules.cljc

```diff
@@ -136,6 +136,12 @@
                [(str ?val) ?str-val]
                [(contains? ?v ?str-val)]))]
 
+   :page-refs
```

Comment:
Unfortunately we can't add new entries to this map without effecting simple queries which we don't want to do. I can fix this when this PR is ready to go